test/caption-stream-test.js

```diff
@@ -216,6 +216,48 @@
     }, 'parsed the third caption');
   });
 
+  test('backspaces are applied to non-displayed memory', function() {
+    var captions = [];
+    cea608Stream.on('data', function(caption) {
+      captions.push(caption);
+    });
+
+    // RCL, resume caption loading
+    cea608Stream.push({ ccData: 0x1420 });
+    // '01'
+    cea608Stream.push({
+      ccData: ('0'.charCodeAt(0) << 8) | '1'.charCodeAt(0)
```

Comment:
My object was to having code such as `('0'.charCodeAt(0) << 8) | '1'.charCodeAt(0)` repeated multiple times. Doing the change you show above is adequate.